### DISCUSSION TITLE
Jittering on third person follow only

### DISCUSSION THREAD

**Discussion by RatWedding** [ramokz/phantom-camera#289] [Question / Help Wanted] [answered]
Hello,

I am encountering an issue I lack the skills to understand.

https://github.com/ramokz/phantom-camera/assets/72044365/46002d82-bb64-48d3-b6e5-e99973443966

for some reason on Third person follow mode only, the camera jitters. This is not the case for Simple for example.

So far it seems the issue should be that movement is calculated on _process instead of _physics_process but the movement script is the default godot script. all movement is done on physics process. I cant seem to find anyone else who had a similar issue that wasnt caused by _process

here is also what the node set up, inspector looks like. in case that helps.
![image](https://github.com/ramokz/phantom-camera/assets/72044365/865a3c10-b924-4380-9e4c-b1c2f0d2f14a)

Thank you

**Comment by ramokz** [comment, marked as answer]
Have you tried following the suggestions in [the FAQ](https://phantom-camera.dev/support/faq#i-m-seeing-jitter-what-can-i-do)?

**Reply by RatWedding** [reply]
Yes, I apologize for missing the FAQ
